opusfilter/filters.py

```diff
@@ -10,6 +10,7 @@
 from langid.langid import LanguageIdentifier, model
 import pycld2
 from bs4 import BeautifulSoup as bs
+import fasttext
```

Comment:
While you have added fasttext to requirements.txt, it's missing from `install_requires` list in setup.py.